src/TransientFETools/TransientFETerms.jl

```diff
@@ -154,3 +155,25 @@ end
 function _push_vector_contribution!(v,r,cellvals::Nothing,cellids)
   nothing
 end
+
+"""
+Alternative constructor for affine operators
+"""
+function TransientAffineFETerm(
+  m::Function,a::Function,b::Function,trian::Triangulation,quad::CellQuadrature)
+  res(t,u,ut,v) = m(t,ut,v) + a(t,u,v) - b(t,v)
+  jac(t,u,ut,du,v) = a(t,du,v)
+  jac_t(t,u,ut,dut,v) = m(t,dut,v)
+  FETerm(res,jac,jac_t,trian,quad)
+end
+
+"""
+Alternative constructor for constant operators
+"""
+function TransientConstantFETerm(
```

Comment:
Linear... it is not. So, I don't have a better name now. It is transient because it involves time derivatives, but the operators are constant. We could change the name `Transient` by something else, I don't know what. We can always do renaming later on